### DISCUSSION TITLE
APPIUM INSPECTOR starts an error,The Xcode version is 15.0; appium is 1.22.3, and the version of the iOS real machine mobile phone is 15.7.2.

### DISCUSSION THREAD

**Discussion by yyeira** [appium/appium#19663] [Q&A] [answered]
[debug] [W3C] Encountered internal error running command: Error: '15.7.2' does not exist in the list of simctl SDKs. Only the following Simulator SDK versions are available on your system: 16.1, 17.0
[debug] [W3C] at Simctl.getDevices (/Users/xx/brew/lib/node_modules/appium/node_modules/node-simctl/lib/subcommands/list.js:171:9)

**Comment by KazuCocoa** [comment]
Per the error message, your Xcode environment can create iOS 16.1 or 17.0 simulators

**Reply by KazuCocoa** [reply, marked as answer]
Btw, please use newer XCUITest driver versions as https://appium.github.io/appium-xcuitest-driver/5.14/setup/ since appium 1.22.3 has no compatibility with Xcode 15

**Reply by yyeira** [reply]
That solves my problem. Thank you very much
